frontend/e2e/xmas-tree-locations.e2e-spec.ts

```diff
@@ -8,14 +8,14 @@ describe('Xmas tree  - Where to Find Your Tree page', () => {
     beforeEach(() => {
       page = new TreesSidebarPage();
       browser.driver.manage().window().setSize(1400, 900);
-      page.navigateTo(3);
+      page.navigateTo('mthood');
       browser.sleep(800);
       page.getTreeLocationLink().click();
       browser.sleep(800);
     });
 
     it('should have a link to the external forest maps-pubs', () => {
-      expect<any>(element(by.id('maps-link-3')).getAttribute('href')).toEqual('https://www.fs.usda.gov/main/mthood/maps-pubs');
+      expect<any>(element(by.id('forest-maps-pubs-link')).getAttribute('href')).toEqual('https://www.fs.usda.gov/main/mthood/maps-pubs');
```

Comment:
I don't know if we want to emulate their `-pubs` suffix, that is not particularly readable.